files/SQLite.props

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU'">
+  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU' And '$(ExcludeSqliteNative)' != 'true' ">
```

Comment:
> project.json will go away for .NET Core projects

Thanks for the update @emgarten. This is going to break the currently-released version, SQLite 3.12.2. We should consider a patch release, 3.12.3 to prepare for this change. (I've just opened https://github.com/natemcmaster/libsqlite3-package/issues/8)